From 34d61e5267c861fc704ac2d3875fd14767880246 Mon Sep 17 00:00:00 2001
From: Richard Laager <rlaager@wiktel.com>
Date: Mon, 4 Feb 2019 00:59:08 -0600
Subject: [PATCH] Use .egg-info for the Python info file

Every one of these on my system is named .egg-info, not .info, and that
was the extension discussed in #503.
---
 pylib/ntp-in.egg-info | 12 ++++++++++++
 pylib/ntp-in.info     | 12 ------------
 pylib/wscript         |  4 ++--
 3 files changed, 14 insertions(+), 14 deletions(-)
 create mode 100644 pylib/ntp-in.egg-info
 delete mode 100644 pylib/ntp-in.info

diff --git a/pylib/ntp-in.egg-info b/pylib/ntp-in.egg-info
new file mode 100644
index 000000000..b672a8431
--- /dev/null
+++ b/pylib/ntp-in.egg-info
@@ -0,0 +1,12 @@
+Metadata-Version: 1.0
+Name: ntpsec
+Version: @NTPSEC_VERSION_EXTENDED@
+Platform: POSIX
+Summary: The NTP reference implementation, refactored
+Home-page: https://ntpsec.org/
+License:  Beerware, BSD-2-Clause, BSD-3-Clause, BSD-4-Clause, CC-BY-4.0, ISC, MIT, NTP
+Description: 
+         The Network Time Protocol suite, refactored
+Author: the NTPsec project
+Author-email: devel@ntpsec.org
+Keywords: 
\ No newline at end of file
diff --git a/pylib/ntp-in.info b/pylib/ntp-in.info
deleted file mode 100644
index b672a8431..000000000
--- a/pylib/ntp-in.info
+++ /dev/null
@@ -1,12 +0,0 @@
-Metadata-Version: 1.0
-Name: ntpsec
-Version: @NTPSEC_VERSION_EXTENDED@
-Platform: POSIX
-Summary: The NTP reference implementation, refactored
-Home-page: https://ntpsec.org/
-License:  Beerware, BSD-2-Clause, BSD-3-Clause, BSD-4-Clause, CC-BY-4.0, ISC, MIT, NTP
-Description: 
-         The Network Time Protocol suite, refactored
-Author: the NTPsec project
-Author-email: devel@ntpsec.org
-Keywords: 
\ No newline at end of file
diff --git a/pylib/wscript b/pylib/wscript
index 0181f8afe..a3af703b1 100644
--- a/pylib/wscript
+++ b/pylib/wscript
@@ -80,10 +80,10 @@ def build(ctx):
     )
 
     # pep241  lay an egg
-    egg = ['ntp-%s.info' % ctx.env.NTPSEC_VERSION]
+    egg = ['ntp-%s.egg-info' % ctx.env.NTPSEC_VERSION]
     ctx(
         features="subst",
-        source=['ntp-in.info'],
+        source=['ntp-in.egg-info'],
         target=egg
     )
     ctx.install_files(ctx.env.PYTHONARCHDIR, egg)
-- 
2.17.1

